api/bash/delete-empty-repos.sh

```diff
@@ -0,0 +1,230 @@
+#/   - Example Output: List all empty repositories
+#/     <organization>:<repository1>
+#/     <organization>:<repository2>
+#/     <organization>:<repository3>
+#/
+#/ PRE-REQUISITES:
+#/ Before running this script, you must create a Personal Access Token (PAT)
+#/ at https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/
+#/ with the permissions <repo> and <admin:org> scopes and <delete_repo>. Read more
+#/ about scopes here: https://developer.github.com/apps/building-oauth-apps/scopes-for-oauth-apps/
+#/
+#/ Once created, you must export your PAT as an environment variable
+#/ named <GITHUB_TOKEN>.
+#/   - Exporting PAT as GITHUB_TOKEN
+#/   $ export GITHUB_TOKEN=abcd1234efg567
+#/
+#/ Additionally you will need to set the $API_ROOT at the top of the script to
+#/ your instance of GitHub Enterprise.
+#/  - _i.e._: https://MyGitHubEnterprise.com/api/v3
+#/
+#/ Finally, you will need to ensure you have installed jq: https://stedolan.github.io/jq/
+#/
+#/ OPTIONS:
+#/ --org
+#/ -o
+#/ When running the tool, this flag sets which organization's repositories you
+#/ want to inspect and delete (if they're empty).
+#/
+#/ --execute
+#/ -e
+#/ When running the tool, this flag will delete every repo listed.
+#/   * _NOTE:_ You should run the script without this option first, verifying
+#/      that you want to delete every repository listed.
+#/
+#/ EXAMPLES:
+#/
+#/   - Lists all empty repositories for the given organization.
+#/   $ bash delete-empty-repos.sh --org=MyOrganization
+#/
+#/   - Deletes all empty repositories for the given organization.
+#/   $ bash delete-empty-repos.sh --org=MyOrganization --execute=TRUE
+#/
+#/ API DOCUMENTATION:
+#/ All documentation can be found at https://developer.github.com/v3/
+
+##########
+# HEADER #
+##########
+
+echo ""
+echo "############################################"
+echo "############################################"
+echo "###                                      ###"
+echo "### Delete Empty Repos from Organization ###"
+echo "###                                      ###"
+echo "############################################"
+echo "############################################"
+echo ""
+
+########
+# VARS #
+########
+API_ROOT="https://<your-domain>/api/v3"
+EXECUTE="FALSE"
+EMPTY_REPO_COUNTER=0
+
+##################################
+# Parse options/flags passed in. #
+##################################
+
+for param in "$@"
+do
+  case $param in
+    -e=*|--execute=*)
+    EXECUTE="${param#*=}"
+    shift
+    ;;
+    -o=*|--org=*)
+    ORG_NAME="${param#*=}"
+    shift
+    ;;
+    *)
+    # unknown option, do nothing
+    ;;
+  esac
+done
+
+#################
+# Verify Inputs #
+#################
+
+# If GITHUB_TOKEN wasn't set in Environment
+if [[ -z ${GITHUB_TOKEN} ]]; then
+  echo "ERROR: GITHUB_TOKEN was not found in your environment. You must export "
+  echo "this token prior to running the script."
+  echo "  Ex: EXPORT GITHUB_TOKEN=abc123def456"
```

Comment:
I believe this should be lowercase `export`:

```
EXPORT GITHUB_TOKEN=abc123def456
-bash: EXPORT: command not found
```

Reply:
Great catch, thanks!